index.html

```diff
@@ -0,0 +1,320 @@
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div>
+        <nav id="navbar">
+            <h1>JS Documentation</h1>
+            <ol>
+                <li><a href="index.html#Introduction" class="nav-link">Introduction</a></li>
+                <li><a href="index.html#1" class="nav-link">What you should already Know</a></li>
+                <li><a href="index.html#2" class="nav-link">JavaScript and Java</a></li>
+                <li><a href="index.html#3" class="nav-link">Hello world</a></li>
+                <li><a href="index.html#4" class="nav-link">Variables</a></li>
+                <li><a href="index.html#5" class="nav-link">Declaring variables</a></li>
+                <li><a href="index.html#6" class="nav-link">Variable scope</a></li>
+                <li><a href="index.html#7" class="nav-link">Global variables</a></li>
+                <li><a href="index.html#8" class="nav-link">Constants</a></li>
+                <li><a href="index.html#9" class="nav-link">Data types</a></li>
+                <li><a href="index.html#10" class="nav-link">if...else statement</a></li>
+                <li><a href="index.html#11" class="nav-link">While statement</a></li>
+                <li><a href="index.html#12" class="nav-link">Function declarations</a></li>
+                <li><a href="index.html#13" class="nav-link">Reference</a></li>
+            </ol>     
+        </nav>
+    </div>
+    <div>
+    <main id="main-doc">
+        <section id="Introduction" >
+            <h1 name ="Introductio">Introduction</h1>
+            <p>JavaScript is a cross-platform, object-oriented scripting language. It is a small and lightweight language. Inside a
+                host environment (for example, a web browser), JavaScript can be connected to the objects of its environment to
+                provide
+                programmatic control over them.</p>
+            <br>
+            
+            <p>JavaScript contains a standard library of objects, such as Array, Date, and Math, and a core set of language elements
+                such as operators, control structures, and statements. Core JavaScript can be extended for a variety of purposes by
+                supplementing it with additional objects; for example:</p>
+            
+            <ul>
+                <li>
+                    <p>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document
+                        Object Model
+                        (DOM). For example, client-side extensions allow an application to place elements on an HTML form and
+                        respond to user
+                        events such as mouse clicks, form input, and page navigation.</p>
+                </li>
+                <br>
```

Comment:
Kindly remove the br tag inside the ul tag as it is not a list item. Please style the li in the CSS instead by adding a margin to add a space between the list items